### ISSUE TITLE
Enhance abscal tests to see if input absolute calibration is recovered

### ISSUE THREAD

**Issue by jsdillon** [HERA-Team/hera_cal#135]
`Test_AbsCal_Funcs` currently has 
```        # make mock data
         abs_gain = 0.5
         TT_phi = np.array([-0.004, 0.006, 0])
         model = odict()
         for i, k in enumerate(data.keys()):
             model[k] = data[k] * np.exp(abs_gain + 1j*np.dot(TT_phi, bls[k]))```

You should be able to run abscal on your mock data and recover these numbers very precisely. This would be a stronger unit test for abscal beyond the current tests that just show that abscal ran and produced arrays of the correct size (but not necessarily the correct value).

**Comment by jsdillon** [comment]
`test_fft_dly` is another example that only tests whether the code runs, not whether it's right.

**Comment by nkern** [comment]
addressed by #140 PR
